edit_book.php

```diff
@@ -0,0 +1,44 @@
+<?php
+include 'database.php';
```

Comment:
Observations: The code does not handle the redirection properly after a successful operation, which can lead to duplicate submissions if the user refreshes the page.

Recommendations:
- **Use Post/Redirect/Get Pattern**: After processing a POST request, redirect the user to a GET request to prevent duplicate submissions.
- **Example**:
  ```php
  if ($bookRepository->updateBook($id, $title, $author, $year)) {
      header("Location: list_books.php?success=1");
      exit();
  }
  ```